### ISSUE TITLE
.dockerignore file can be left with a `dags/` entry if the `astro deploy` command does not run properly

### ISSUE THREAD

**Issue by Slanman3755** [astronomer/astro-cli#1144]
**Describe the bug**

The `.dockerignore` file can be left with a `dags/` entry if the `astro deploy` command does not run properly. This itssue can be easily recreated by manually terminating an `astro deploy` during the docker build. If this change was to go undetected it would cause future DAG deployments to have no DAGs, thus breaking a deployment.

original `.dockerignore` file
```
astro
.git
.env
airflow_settings.yaml
logs/
```

`.dockerignore` after bad `astro deploy`
```
astro
.git
.env
airflow_settings.yaml
logs/

dags/
```

**What CLI Version did you experience this bug?**
Astro CLI Version: 1.10.0

**This CLI bug is related to which Astronomer Platform?**

- [ ] Astro
- [ ] Software
- [x] None/Unknown

**What Operating System is the above CLI installed on?**
macOS Monterey 12.5.1

**🪜 Steps To Reproduce**
```
mkdir hello-astro && cd hello-astro
astro dev init
# inspect .dockerignore file
cat .dockerignore
astro deploy
# ctrl-c while docker image is building
# inspect .dockerignore file again and observe addition of "dags/"
cat .dockerignore
```

**📸 Screenshots**
N/A

**Comment by prasadrayudu** [comment]
Zendesk ticket for more info : [17822](https://astronomer.zendesk.com/agent/tickets/17822)

**Comment by kushalmalani** [comment]
Fix is out in 1.13.1. 
`brew install astronomer/tap/astro@1.13.1`
